user/UserController.js

```diff
@@ -1,69 +1,31 @@
-var express = require('express');
-var router = express.Router();
-var bodyParser = require('body-parser');
+const express = require('express');
+const bodyParser = require('body-parser');
+const ScoutUser = require('../data/models/ScoutUser');
+const Database = require('../data/database');
+const database = new Database();
 
+const router = express.Router();
 router.use(bodyParser.urlencoded({ extended: true }));
 router.use(bodyParser.json());
-var User = require('./User');
 
-// CREATES A NEW USER
-router.post('/', function (req, res) {
-  User.create({
-    name : req.body.name,
-    email : req.body.email,
-    password : req.body.password
-  }, 
-  function (err, user) {
-    if (err) {
-      return res.status(500)
-        .send('There was a problem adding to the database.');
-    }
-    res.status(200).send(user);
-  });
+router.get('/scoutusers', async function(req, res) {
+  try {
+    const articles = await ScoutUser.scan().exec();
+    res.send(articles);
+  } catch (err) {
+    console.error(err);
+    res.sendStatus(500);
+  }
 });
 
-// RETURNS ALL THE USERS IN THE DATABASE
-router.get('/', function (req, res) {
-  User.find({}, function (err, users) {
-    if (err) {
-      return res.status(500).send('There was a problem finding users.');
-    }
-    res.status(200).send(users);
-  });
+router.post('/scoutusers', async function(req, res) {
```

Comment:
(This route will be used by the scout-oauth-access-token-lambda to create the ScoutUser rather than the direct connection it has to Mongo in the current codebase. PR coming soon.)